src/pkgcheck/checks/codingstyle.py

```diff
@@ -1397,3 +1397,43 @@ def feed(self, pkg):
                         lineno, _colno = node.start_point
                         yield GlobDistdir(line=pkg.node_str(node), lineno=lineno + 1, pkg=pkg)
                         break
+
+
+class VariableShadowed(results.LinesResult, results.Warning):
+    """Variable is shadowed or repeatedly declared. This is a possible typo."""
+
+    def __init__(self, var_name, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.var_name = var_name
+
+    @property
+    def desc(self):
+        return f"possible variable {self.var_name!r} is shadowed, {self.lines_str}"
```

Comment:
```suggestion
        return f"Variable {self.var_name!r} appears to be shadowed, {self.lines_str}"
```
or
```suggestion
        return f"Variable {self.var_name!r} is shadowed, {self.lines_str}"
```
or
```suggestion
        return f"Variable {self.var_name!r} may be shadowed, {self.lines_str}"
```

Reply:
I think I like the option 2 or the last one?

Reply:
I think the last one is simpler sounding between the two.